consensus/transition.go

```diff
@@ -53,6 +53,7 @@ type Transition interface {
 }
 
 // TimedOut waiting for some other external event.
+// FIXME: TimedOut should probably have Height and Round
```

Comment:
We'll come back to this one. I am not quite convinced about this, one way or another.